MSCollectionViewCalendarLayout/MSCollectionViewCalendarLayout.m

```diff
@@ -452,7 +452,8 @@ - (void)prepareVerticalTileSectionLayoutForSections:(NSIndexSet *)sectionIndexes
         NSDateComponents *currentDay = [self dayForSection:section];
         NSDateComponents *currentTimeDateComponents = [self currentTimeDateComponents];
         // The current time is within this section's day
-        if ((currentTimeDateComponents.day == currentDay.day) && (currentTimeDateComponents.hour >= earliestHour) && (currentTimeDateComponents.hour < latestHour)) {
+		BOOL isSameDay = (currentTimeDateComponents.year == currentDay.year) && (currentTimeDateComponents.month == currentDay.month) && (currentTimeDateComponents.day == currentDay.day);
```

Comment:
Is this the only place where the fix is needed? Would it make sense for ```isSameDay``` it be put into a separate method?